### ISSUE TITLE
[Low] Show info buttons should contain unique info

### ISSUE THREAD

**Issue by daniel-montalvo** [w3c/wai-course-list#129]
If you use NVDA, JAWS, and VoiceOver list of buttons, these are all shown as "Show info". If you use regular tab navigation, you get info from context.

For a better screen reader user experience, they should be "Show info [about format, scheduling, etc]. And that info should be visually hidden.
